src/jit/jitconfigvalues.h

```diff
@@ -206,6 +205,10 @@ CONFIG_INTEGER(JitEnableNoWayAssert, W("JitEnableNoWayAssert"), 0)
 CONFIG_INTEGER(JitEnableNoWayAssert, W("JitEnableNoWayAssert"), 1)
 #endif // !defined(DEBUG) && !defined(_DEBUG)
 
+//#if MEASURE_MEM_ALLOC -- macro not always available here due to #include ordering
```

Comment:
Anyone else with an opinion? You know what I think since I went to the trouble of adding the comment ;).